PovioKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'PovioKit'
-  s.version          = '0.3.0'
+  s.version          = '0.3.1'
```

Comment:
We'll bump the spec version when we prepare a release branch. Please revert it.

Reply:
Reverted.